js/scripts.js

```diff
@@ -1 +1,158 @@
+}																							// Math.floor zaokrągla liczby zmiennoprzecinkowe w dół do całkowitej liczby (tu: 0, 1 , 2)
+																							// dzięki temu zwracana wartość funkcji to zawsze jedna z 3 opcji!
+
+
+
+function playerPick(playerPick) {															// funkcja wyboru dla gracza i komputera - w linii 35-37 przypisaliśmy nasłuch na buttony odpowiedzialne za wybór opcji w grze
+	var computerPick = getComputerPick();													//w związku z tym kliknięcie danej opcji powoduje odpalenie funkcji playerPick 
+	playerPickElem.innerHTML = playerPick;													//z parametrem playerPick (string) wynikającym z kodu w linii 35-37 - 'rock', 'paper', 'scissors'
+	computerPickElem.innerHTML = computerPick;
+	checkRoundWinner(playerPick, computerPick);												// 1. tworzymy zmienną computerPick, a jej wartością jest wynik funkcji losujęcej dla komputera
+}																							// 2. w tablicy wyników zmieniamy kod html, na wartość parametru playerPick(linia 35-37)
+																							// 3. w tablicy wyników zmieniamy kod html, na wartość wynikającą z funkcji getComputerPick (zmiennej computerPick)
+																							// 4. odpalamy funkcję sprawdzającą kto wygrał w grze checkRoundWinner z parametrami równymi zmiennym playerPick i computerPick!
+
+
+function checkRoundWinner(playerPick, computerPick) {										// Funkcja sprawdzająca wygraną w grze przyjmująca jako parametry 2 zmienne 
+	playerResultElem.innerHTML = computerResultElem.innerHTML = '';							// 1. Ustala wartość wyniku na ' ' (zmienia kod HTML)
+	 //var winnerIs = 'player';																// 2. W zależności od parametrów wejściowych zmienia wynik:
+	if (playerPick == computerPick) {														// 3. Poprzez analizę przypadków, porównuje wartość parametrów playerPick i ComputerPick
+		setResult('noone'); // winnerIs = 'noone'; // remis														  
+	} else if (																				
+		(computerPick == 'rock' &&  playerPick == 'scissors') ||
+		(computerPick == 'scissors' &&  playerPick == 'paper') ||
+		(computerPick == 'paper' &&  playerPick == 'rock')) {
+		setResult('computer'); //winnerIs = 'computer';
+	}
+	else {
+	setResult ('player');
+	}
+}
+
+function setGamePoints() {																	// funkcja zmieniająca wynik w tablicy wyników
+	playerPointsElem.innerHTML = player.score;
+	computerPointsElem.innerHTML = computer.score;
+	if ((computer.score === 10) || (player.score === 10)) {
+		var playerLooser = document.getElementById('js-winnerLooser');
+		if (computer.score === 10) {
+			playerLooser.innerHTML = "Przegrałeś!";
+		}
+		if (player.score === 10) {
+			playerLooser.innerHTML = "Wygrałeś!";
+		}
+		gameState = 'ended';
+		setGameElements();
+	} 
+}
+
+function setResult(winnerIs){
+	if (winnerIs == 'player') {
+		playerResultElem.innerHTML = "Win!";
+		player.score++;
+		setGamePoints();
+	} else if (winnerIs == 'computer') {
+		computerResultElem.innerHTML = "Win!";
+		computer.score++;
+		setGamePoints();
+	}
```

Comment:
zamiast przekazywać do tej funkcji stringa `winnerIs` to przekaż element html i obiekt do aktualizacji (np playerResultElem i player). Dzięki temu pozbędziesz się DRY.

Jeśli będzie remis to po prostu nie wywołuj tej funkcji bo tam nie ma w sumie co robić (nie podbijamy nikomu wyniku rundy)